.github/workflows/build_wheels_linux.yml

```diff
@@ -1,13 +1,180 @@
+      gpu_arch_version: "cu102"
+  binary_linux_wheel_py310_rocm50:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-rocm:5.0"
+      python_version: "3.10"
+      gpu_arch_version: "rocm5.0"
+  binary_linux_wheel_py38_cu113:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda113"
+      python_version: "3.8"
+      gpu_arch_version: "cu113"
+  binary_linux_wheel_py37_rocm511:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-rocm:5.1.1"
+      python_version: "3.7"
+      gpu_arch_version: "rocm5.1.1"
+  binary_linux_wheel_py37_cu116:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda116"
+      python_version: "3.7"
+      gpu_arch_version: "cu116"
+  binary_linux_wheel_py39_cu102:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda102"
+      python_version: "3.9"
+      gpu_arch_version: "cu102"
+  binary_linux_wheel_py38_cpu:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda102"
+      python_version: "3.8"
+      gpu_arch_version: "cpu"
+  binary_linux_wheel_py37_rocm50:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-rocm:5.0"
+      python_version: "3.7"
+      gpu_arch_version: "rocm5.0"
+  binary_linux_wheel_py37_cpu:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda102"
+      python_version: "3.7"
+      gpu_arch_version: "cpu"
+  binary_linux_wheel_py38_rocm50:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-rocm:5.0"
+      python_version: "3.8"
+      gpu_arch_version: "rocm5.0"
+  binary_linux_wheel_py310_cu113:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda113"
+      python_version: "3.10"
+      gpu_arch_version: "cu113"
+  binary_linux_wheel_py310_cpu:
+    uses: pytorch/test-infra/.github/workflows/build_wheels_linux_reusable.yml@main
+    with:
+      repository: "audio"
+      docker_image: "pytorch/manylinux-cuda102"
+      python_version: "3.10"
+      gpu_arch_version: "cpu"
```

Comment:
Just a general note: Apart from cpu builds using the cuda-102 image, there is a fairly straightforward mapping between gpu_arch_version and docker_image. So the matrix strategy can still work with simple if-else logic. Either way, let's merge this in first (and do the same for conda), and we can do the cleanup later.

Reply:
thank you. Will explore the if-else logic afterwards. BTW, can you please merge this PR? I do not seem to have permission.